core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1058,45 +1044,6 @@ public enum Property {
           + "constraint.",
       "2.0.0"),
 
-  // VFS ClassLoader properties
-
-  // this property shouldn't be used directly; it exists solely to document the default value
-  // defined by its use in AccumuloVFSClassLoader when generating the property documentation
-  @Deprecated(since = "2.1.0", forRemoval = true)
-  VFS_CLASSLOADER_SYSTEM_CLASSPATH_PROPERTY(
-      org.apache.accumulo.start.classloader.vfs.AccumuloVFSClassLoader.VFS_CLASSLOADER_SYSTEM_CLASSPATH_PROPERTY,
-      "", PropertyType.STRING,
-      "Configuration for a system level vfs classloader. Accumulo jar can be"
-          + " configured here and loaded out of HDFS.",
-      "1.5.0"),
-  @Deprecated(since = "2.1.0", forRemoval = true)
-  VFS_CONTEXT_CLASSPATH_PROPERTY(
```

Comment:
I think we need a property like this, maybe called GENERAL_CONTEXT_CLASSPATH_PREFIX, so that administrators/users can define the classpaths for a context. Unless you accounted for this somewhere else, it's possible I missed it. The current way users interact with this is to define named classloader context and their paths, for example:

```
general.vfs.context.classpath.<contextName>=<some classpath>
```

Then, in supported places in the code, they can pass in the <contextName> to use their custom supplied classes.

Reply:
@cshannon reminded me that this is not needed based on earlier discussions. The context property set on the table will contain a comma separated list of URIs.